tests/config/configs.py

```diff
@@ -0,0 +1,89 @@
+from abc import ABC
+from enum import Enum
+from typing import Annotated, Any, List, Literal
+
+from pydantic import BaseModel, GetCoreSchemaHandler, validator
+from pydantic_core import core_schema
+
+from tests.config.hierarchical_instantiation.components import ComponentTypes, ComponentVWIF
```

Comment:
This should be `from tests.config.components import ComponentTypes, ComponentVWIF`